pkg/cmd/runsGet.go

```diff
@@ -27,13 +27,14 @@ var (
 
 	// Variables set by cobra's command-line parsing.
 	runName            string
+	age                string
 	outputFormatString string
 )
 
 func init() {
 	runsGetCmd.PersistentFlags().StringVar(&runName, "name", "", "the name of the test run we want information about")
+	runsGetCmd.PersistentFlags().StringVar(&age, "age", "", "the age of the test run(s) we want information about. Supported formats are: --age FROM, --age FROM:TO. Supported units are days, weeks and hours. Examples: --age 1d, --age 6h:1h")
```

Comment:
Better wording: ?

Supported formats are: 'FROM' or 'FROM:TO', where FROM and TO are each ages, made up of an integer and a time-unit qualifier. Supported time-units are days, weeks and hours. Examples: --age 1d, --age 6h:1h")

are 'days' 'hours' also supported ?